packages/core/src/embeddings/types.ts

```diff
@@ -10,7 +10,13 @@ export enum SimilarityType {
   EUCLIDEAN = "euclidean",
 }
 
+type Embedding = number[];
```

Comment:
good idea using a type for this, but then we should introduce it for all embeddings